lib/std/array_list.zig

```diff
@@ -283,13 +299,9 @@ pub fn ArrayListAligned(comptime T: type, comptime alignment: ?u29) type {
         /// This operation is O(N).
         /// This preserves item order. Use `swapRemove` if order preservation is not important.
         pub fn orderedRemove(self: *Self, i: usize) T {
-            const newlen = self.items.len - 1;
-            if (newlen == i) return self.pop();
-
+            const empty: []T = &.{};
```

Comment:
Is there a better way to do this?

Reply:
If I'm not mistaken you can just get rid of that line and do `self.replaceRangeAssumeCapacity(i, 1, &.{});` below?